### ISSUE TITLE
Latest  working Speckle connector for Tekla is ver. 2.13.3! Every never version of it crashes. Verified with TS2020 ...TS2023.

### ISSUE THREAD

**Issue by bau-tech** [specklesystems/speckle-sharp#2688]
## Prerequisites

- [ ] I read the [contribution guidelines](https://github.com/specklesystems/speckle-server/blob/main/CONTRIBUTING.md)
- [ ] I checked the [documentation](https://speckle.guide/) and found no answer.
- [ ] I checked [existing issues](../issues?q=is%3Aissue) and found no similar issue. 
- [ ] I checked the [community forum](https://speckle.community/) for related discussions and found no answer.
- [ ] I'm reporting the issue to the correct repository (see also [speckle-server](https://github.com/specklesystems/speckle-server), [speckle-sharp](https://github.com/specklesystems/speckle-sharp), [specklepy](https://github.com/specklesystems/specklepy), [speckle-docs](https://github.com/specklesystems/speckle-docs), and [others](https://github.com/orgs/specklesystems/repositories))

## System Info

If applicable, please fill in the below details - they help a lot!

**Comment by teocomi** [comment]
Hello, have you tried with version 2.14.4 as well?

**Comment by bau-tech** [comment]
Yes I have tried every single version also with the latest 2.14.4 and with all 4 TS2020-TS2023.   Starting the connector will also crash Tekla.



Von: Matteo ***@***.***>
Gesendet: Donnerstag, 22. Juni 2023 10:12
An: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Betreff: Re: [specklesystems/speckle-sharp] Latest working Speckle connector for Tekla is ver. 2.13.3! Every never version of it crashes. Verified with TS2020 ...TS2023. (Issue #2688)


Hello, have you tried with version 2.14.4 as well?

**Comment by teocomi** [comment]
Hey @bau-tech can you please try with our latest alpha?
![image](https://github.com/specklesystems/speckle-sharp/assets/2679513/55ea50a7-ade9-41a5-9560-c62f3eedc5ce)

**Comment by bau-tech** [comment]
Hi everyone,

Now I have tested  the 2.15alpha5 version with the following results:

Tekla Structures 2020 -> not working. UI of the Speckle connector does not show up.,but it does not crash Tekla anymore.
Tekla Structures 2021 ->is working!
Tekla Structures 2022 ->is working!
Tekla Structures 2023 ->is working!

Thanks

Gesendet von Mail<https://go.microsoft.com/fwlink/?LinkId=550986> für Windows

Von: Matteo ***@***.***>
Gesendet: Dienstag, 4. Juli 2023 17:34
An: ***@***.***>
Cc: ***@***.***>; ***@***.***>
Betreff: Re: [specklesystems/speckle-sharp] Latest working Speckle connector for Tekla is ver. 2.13.3! Every never version of it crashes. Verified with TS2020 ...TS2023. (Issue #2688)


Hey @bau-tech<https://github.com/bau-tech> can you please try with our latest alpha?
[image]<https://user-images.githubusercontent.com/2679513/250945407-55ea50a7-ade9-41a5-9560-c62f3eedc5ce.png>

**Comment by teocomi** [comment]
Thanks so much for testing across all versions :)

I'll investigate what's up with 2020 - how bad would it be if we dropped support for it? Are you relying on it much?
